salt/modules/rpm.py

```diff
@@ -424,30 +419,56 @@ def info(*packages):
     .. code-block:: bash
 
         salt '*' lowpkg.info apache2 bash
+        salt '*' lowpkg.info apache2 bash attr=version
+        salt '*' lowpkg.info apache2 bash attr=version,build_date_iso,size
     '''
 
     cmd = packages and "rpm -q {0}".format(' '.join(packages)) or "rpm -qa"
 
-    # Locale needs to be en_US instead of C, because RPM otherwise will yank the timezone from the timestamps
-    call = __salt__['cmd.run_all'](cmd + (" --queryformat 'Name: %{NAME}\n"
-                                                          "Relocations: %|PREFIXES?{[%{PREFIXES} ]}:{(not relocatable)}|\n"
-                                                          "Version: %{VERSION}\n"
-                                                          "Vendor: %{VENDOR}\n"
-                                                          "Release: %{RELEASE}\n"
-                                                          "Build Date: %{BUILDTIME:date}\n"
-                                                          "Install Date: %|INSTALLTIME?{%{INSTALLTIME:date}}:{(not installed)}|\n"
-                                                          "Build Host: %{BUILDHOST}\n"
-                                                          "Group: %{GROUP}\n"
-                                                          "Source RPM: %{SOURCERPM}\n"
-                                                          "Size: %{LONGSIZE}\n"
-                                                          "%|LICENSE?{License: %{LICENSE}\n}|"
-                                                          "Signature: %|DSAHEADER?{%{DSAHEADER:pgpsig}}:{%|RSAHEADER?{%{RSAHEADER:pgpsig}}:{%|SIGGPG?{%{SIGGPG:pgpsig}}:{%|SIGPGP?{%{SIGPGP:pgpsig}}:{(none)}|}|}|}|\n"
-                                                          "%|PACKAGER?{Packager: %{PACKAGER}\n}|"
-                                                          "%|URL?{URL: %{URL}\n}|"
-                                                          "Summary: %{SUMMARY}\n"
-                                                          "Description:\n%{DESCRIPTION}\n"
-                                                          "-----\n'"),
-                                   output_loglevel='trace', env={'LC_ALL': 'en_US', 'TZ': 'UTC'}, clean_env=True)
+    # Construct query format
+    attr_map = {
```

Comment:
I very much like the ability to ask for `{build,install}_date` vs `{build,install}_date_time_t` you had before.